docs/netlify_deploy.md

```diff
@@ -0,0 +1,38 @@
+# Netlify deploy process
+
+We use Netlify to deploy clayui.com and a version of Storybook, the deployment process is very simple, the build happens in the service itself so we don't have a Github Actions configuration.
+
+A new version of clayui.com and storybook are triggered in different scenarios.
+
+-   **Deploy Previews** - every new PR a new build is made to show the changes made
+-   **next.clayui.com** - Represents changes that are in the master branch
+-   **next.storybook.clayui.com** - Represents changes that are in the master branch
+-   **clayui.com** - Represents changes that are in the stable branch
+-   **storybook.clayui.com** - Represents changes that are in the stable branch
+
+## Deploy Previews
+
+All PR triggers a new build for clayui.com and storybook and generates a preview URL that allows you to preview the changes without having to download the PR locally and test.
```

Comment:
```suggestion
All pull requests trigger a new build of clayui.com and storybook and generate a preview URL that allows you to view the changes without having to download the changes locally.
```

PR could mean "public relations" or something else, I think it's better to be specific and use complete words.